programs/cw-ics08-wasm-eth/src/msg.rs

```diff
@@ -63,7 +64,18 @@ pub enum QueryMsg {
 #[cw_serde]
 pub struct MigrateMsg {
     /// The optional instantiate msg to re-initialize the client
-    pub instantiate_msg: Option<InstantiateMsg>,
+    pub migration: Migration,
```

Comment:
You could have this enum be the entire `MigrateMsg` instead of a field. I'm also ok with this, idk if you prefer this

Reply:
No do whatever is rust convention

Reply:
keeping as-is for now